packages/client/bin/utils.ts

```diff
@@ -196,6 +196,16 @@ export function getArgs(): ClientOpts {
         boolean: true,
         default: true,
       })
+      .option('rpcEthMaxPayload', {
+        describe: 'Define max JSON payload size for eth/debug RPC requests',
+        string: true,
+        default: '5mb',
+      })
+      .option('rpcEngineMaxPayload', {
+        describe: 'Define max JSON payload size for engine RPC requests',
+        string: true,
+        default: '15mb',
```

Comment:
Could you point this to `RPC_ENGINE_MAXPAYLOAD_DEFAULT`? Also for `rpcEthMaxPayload` :smile: :+1: